core/src/main/scala/cats/instances/try.scala

```diff
@@ -141,6 +138,14 @@ trait TryInstances extends TryInstances1 {
       override def catchNonFatal[A](a: => A)(implicit ev: Throwable <:< Throwable): Try[A] = Try(a)
 
       override def catchNonFatalEval[A](a: Eval[A])(implicit ev: Throwable <:< Throwable): Try[A] = Try(a.value)
+
+      private val successUnit: Try[Unit] = Success(())
```

Comment:
```suggestion
      private[this] val successUnit: Try[Unit] = Success(())
```

`private[this]` is much faster than `private`